import java.util.Scanner;

class Login {
    
    static void login(String password) throws Exception {
        String correctPassword = "12345";

        if (!password.equals(correctPassword)) {
            throw new Exception("Invalid Password!");
        }

        System.out.println("Login Successful!");
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        try {
            System.out.print("Enter Password: ");
            String password = sc.nextLine();

            login(password);
        }
        catch (Exception e) {
            System.out.println("Error: " + e.getMessage());
        }
        finally {
            System.out.println("Login process completed.");
        }

        sc.close();
    }
}